conf/script/build-ha-sspl

```diff
@@ -159,7 +159,7 @@ consul_config() {
     ssh $rnode $cmd
 
     echo 'Reloading Consul...'
-    /opt/seagate/cortx/hare/bin/consul reload
+    /usr/bin/consul reload
```

Comment:
@mssawant 
is there any change for 
/var/lib/hare/consul-server-c1-conf/ /var/lib/hare/consul-server-c2-conf/

consul watcher

Reply:
I think no, but I would let @mssawant confirm.